tests/suites/os/tests/bluetooth/index.js

```diff
@@ -3,6 +3,16 @@ const exec = Bluebird.promisify(require('child_process').exec);
 
 module.exports = {
 	title: 'Bluetooth tests',
+	workerContract: {
+		type: 'object',
+		required: ['workerType'],
+		properties: {
+			workerType: {
+				type: 'string',
+				const: 'qemu'
```

Comment:
I'm confused, why do we set the worker type to qemu here?

Reply:
Very good catch @klutchell the worker type indeed need to be testbot_hat here

Reply:
But, weird how did the tests pass.

Reply:
The tests failed before the suite actually started @vipulgupta2048: https://github.com/balena-os/leviathan/pull/604 this just merged and the qemu tests use the latest revision of core

Reply:
That being true. I don't think we should let things like this go. I might have a potential solution to catch these too and fail the job.